app/views/landing/index.html.slim

```diff
@@ -1,25 +1,33 @@
 .logo-section
-  = image_tag('landing/phone.png')
+  = image_tag 'landing/phone.png'
   .vertical-separator
-  = image_tag('landing/tcg_logo.png')
+  = image_tag 'landing/tcg_logo.png'
 
 .features-section
   .feature
-    = image_tag('landing/ic_list.png', class:'feature-img')
-    h3.feature-title Keep track of your money
-    p.feature-p See all your debts and all your friends debts towards you.
+    = image_tag 'landing/ic_list.png', class:'feature-img'
+    h3.feature-title
+      | Keep track of your money
+    p.feature-p
+      | See all your debts and all your friends debts towards you.
 
   .feature
-    = image_tag('landing/ic_reminder.png', class:'feature-img')
-    h3.feature-title Generate reminders
-    p.feature-p Get reminded of who owes you money or who you owe money to
+    = image_tag 'landing/ic_reminder.png', class:'feature-img'
+    h3.feature-title
+      | Generate reminders
+    p.feature-p
+      | Get reminded of who owes you money or who you owe money to
 
   .feature
-    = image_tag('landing/ic_safe.png', class:'feature-img')
-    h3.feature-title No internet needed
-    p.feature-p All your debts are kept safe in your phone
+    = image_tag 'landing/ic_safe.png', class:'feature-img'
+    h3.feature-title
+      | No internet needed
+    p.feature-p
+      | All your debts are kept safe in your phone
 
 form.footer action="#"
-  h3.form-text Get info on new updates
+  h3.form-text
+    | Get info on new updates
   input.footer-input type="text"
-  button.submit-button.form-text Subscribe
+  button.submit-button.form-text
```

Comment:
Being this a submit button, you need to add  `type="submit"` so to let the form know